### PR TITLE
Safari10

### PR DESCRIPTION
**Pull request by chicoxyzzy**
- add Safari 10
- fix es-next features support for Safari TP and WebKit Nightly

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -305,6 +305,11 @@ exports.browsers = {
     short: 'SF 9',
     obsolete: false,
   },
+  safari10: {
+    full: 'Safari 10',
+    short: 'SF 10',
+    obsolete: false,
```
Comment (zloirock) [inline data-esnext.js]: `unstable: true`

**Comment by zloirock** [comment]
I don't see all es6 diff, but results are incorrect. For example, you enabled TCO for Chrome 52, but it's available only with the flag - #790

**Comment by zloirock** [comment]
Also, can you avoid merge commits in PRs in the future? It complicates checking and pollutes the repo.

**Comment by chicoxyzzy** [comment]
commit with chrome was mistakenly taken from another branch (it was in my [`gh-pages`](https://github.com/kangax/compat-table/pull/790) from which I created new branch). Removed it back.

> Also, can you avoid merge commits in PRs in the future? It complicates checking and pollutes the repo.

hmm what if I squash all commits in this PR?

**Comment by chicoxyzzy** [comment]
> hmm what if I squash all commits in this PR?

realized it's bad idea

**Comment by ljharb** [comment]
Rebasing down to the proper number of commits such that each commit represents an atomic, conceptual change, is a great idea :-)

**Comment by chicoxyzzy** [comment]
fixed

**Pull request merged**